ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -367,7 +367,12 @@ void WaylandWindow::Configure(void* data,
 
   window->ResetWindowStates();
   uint32_t* p;
-  wl_array_for_each(p, states) {
+  // wl_array_for_each has a bug in upstream. It tries to assign void* to
+  // uint32_t *, which is not allowed in C++. Explicit cast should be performed.
+  // We can't modify wayland-util.h, because it is fetched with gclient sync.
+  // Thus, use own loop.
+  for (p = (uint32_t*)states->data;
```

Comment:
And, yeah, seems like nobody is using wl_array_for_each, that's why nobody has faced this kind of error yet.